deployments/pulumi/docs/schema.md

```
@@ -0,0 +1,246 @@
+# jsonschema-markdown
+
+JSON Schema missing a description, provide it using the `description` key in the root of the JSON document.
+
+### Type: `object(?)`
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)